news/6741.feature

```diff
@@ -0,0 +1 @@
+Add a subcommand ``list-files`` to ``pip config`` to list available configuration files and their location
```

Comment:
This should use the correct command name.

Reply:
Thanks, also do you have any suggestions on how to show the debug options defined as environment variables, as to what should the key look like in that case.?

I feel that we should be using the [PIP_<UPPER_LONG_NAME>](https://pip.pypa.io/en/stable/user_guide/#environment-variables) for it.

Reply:
The command has been renamed in https://github.com/pypa/pip/pull/8096/commits/554a8451703d6d16ae052c20cd85c2dfbf995d49 to `debug` as well.

Reply:
Okay, the output for `env` and `env-var` is now 

```
env_var:
  PIP_CONFIG_FILE='/Users/devesh/Desktop/pip.config'
  PIP_DEFAULT_TIMEOUT='60'
env:
  /Users/devesh/Desktop/pip.config, exists: True 
    global.timeout: 60 
    freeze.timeout: 10 
```

added in https://github.com/pypa/pip/pull/8096/commits/30bc04ac2ad4c5ddfdc4f13c1271b1d69dfdcd94